rpcs3/Emu/Io/usio.h

```diff
@@ -21,6 +21,10 @@ class usb_device_usio : public usb_device_emulated
 	void usio_read(u8 channel, u16 reg, u16 size);
 
 private:
+	bool test_sw_on = false;
+	bool last_test_sw = false;
+	bool last_coin_sw = false;
```

Comment:
please write out sw. it's not necessary to abbreviate this.
Also, there is not a single comment that explains this new logic

Reply:
Done.